frontend/src/js/external-forms/form-components/DropzoneBetweenElements.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from "@emotion/styled";
+import { useState } from "react";
+import { DropTargetMonitor, useDrop } from "react-dnd";
+
+import { PossibleDroppableObject } from "../../ui-components/Dropzone";
+
+interface Props<DroppableObject> {
+  onDrop: (item: DroppableObject, monitor: DropTargetMonitor) => void;
+  acceptedDropTypes: string[];
+}
+
+const Root = styled("div")<{
+  height: number;
+}>`
+  width: 100%;
+  left: 0;
+  top: -15px;
+  height: ${({ height }) => height + 40}px;
+  right: 0;
+  position: relative;
+  border-radius: ${({ theme }) => theme.borderRadius};
+`;
+
+const Expander = styled("div")<{
+  height: number;
+}>`
+  overflow: hidden;
+  margin-top: ${({ height }) => -height}px;
```

Comment:
Let's avoid setting very dynamic numeric values in styled components (to avoid recalculating classnames) and let's use inline styles instead